app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        mv, source, destination = command.split()
+        directory = os.path.dirname(destination)
+        new_destination = os.path.join(
+            directory, os.path.basename(destination)
+        )
+        if mv == "mv":
+            if directory:
+                os.makedirs(directory, exist_ok=True)
+
+            shutil.move(source, new_destination)
```

Comment:
```suggestion
def move_file(command: str) -> None:
    command = command.split()
    if len(command) != 3 or command[0] != "mv":
        raise Exception(
            "Invalid command; must be format: <mv> <source_file> <destination>"
        )

    _, source, destination = command
    directory = os.path.dirname(destination)
    if directory:
        os.makedirs(directory, exist_ok=True)

    shutil.move(source, destination)
```

1. do not use the `command.split()` twice
2. If something goes wrong, it's better to throw an error so that the user understands